lib/go-tc/tovalidate/rules.go

```diff
@@ -217,3 +217,14 @@ func StringIsValidFloat() *validation.StringRule {
 		return err == nil && !math.IsNaN(validated)
 	}, "must be a valid float")
 }
+
+// IsValidIPorCIDR returns whether the input is a valid IP address or CIDR notation subnet
+func IsValidIPorCIDR(input string) bool {
```

Comment:
nit: The name should be `IsValidIPOrCIDR`, instead of `IsValidIPorCIDR`
Note that the `O` needs to be capitalized